website/frontend/src/components/blocks/Header/children/Menu/Menu.tsx

```diff
@@ -12,6 +13,10 @@ type Props = {
 const Menu = ({ openSidebar }: Props) => {
   const { isApiReady } = useSelector((state: RootState) => state.api);
 
+  const setClassName = (isActive: boolean) => {
+    return isActive ? clsx(styles.link, styles.active) : styles.link;
+  };
+
```

Comment:
```suggestion
  const getClassName = ({ isActive }: ClassNameProps) => clsx(styles.link, isActive && styles.active);
```
I believe`setClassName` is not appropriate name since we're getting className, not setting it.
Also you can write it simpler